cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/BufferingTable.java

```diff
@@ -109,14 +110,16 @@ public BufferingTable(String name) {
    * @param name table name
    */
   public BufferingTable(String name, ConflictDetection level) {
-    this(name, level, false);
+    this(name, level, false, null, null);
   }
 
   /**
    * Creates an instance of {@link BufferingTable}.
    * @param name table name
    */
-  public BufferingTable(String name, ConflictDetection level, boolean enableReadlessIncrements) {
+  public BufferingTable(String name, ConflictDetection level, boolean enableReadlessIncrements,
+                        Schema schema, String schemaRowField) {
```

Comment:
It would be a good idea to add `@Nullable` annotation on `schema` and `schemaRowField`